exporter/awsxrayexporter/translator/cause.go

```diff
@@ -27,6 +28,10 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/awsxray"
 )
 
+var (
+	javascriptStackTracePrefixRegex = regexp.MustCompile(`\ *at (.*?)`)
```

Comment:
Instead of regex can you do just a prefix check?

Reply:
The prefix of a v8 stack trace is of the format '    at ...', I can't find the total number of spaces of each prefix which is why I chose to use regex. Do you have any suggestions for how I could use a prefix check?

Reply:
Why do you suspect it may have a different number of spaces than 4? I couldn't find any indication of that in the docs so feel the strict parsing could be fine.

Otherwise let's at least add tests with different indent - ideally they're from real stack traces though.

Reply:
I don't have any real suspicions other than just being strictly conservative - I changed it to strictly parse with 4 spaces and removed the regex.